setup.py

```diff
@@ -41,10 +41,9 @@ def package_files(directory):
     "databricks-cli>=0.8.7",
     "entrypoints",
     "gitpython>=2.1.0",
-    "numpy",
-    "pandas",
     "pyyaml",
     "protobuf>=3.6.0",
+    "pytz",
```

Comment:
Is this library necessary? I don't see it imported elsewhere in the PR code and I was under the impression that the only datetime functionality used by Pandas is leveraged in `search_runs`, which we're working around by providing an alternative list-based output format.

Reply:
mlflow/utils/time_utils.py:from pytz import reference

Reply:
Ah, cool. This was pulled in transitively via pandas before?

Reply:
yup